src/rules/helpers/astHelpers.ts

```
@@ -32,114 +32,112 @@ const {
- * @param {FunctionDeclaration | ArrowFunctionExpression} node Function declaration
- * @param {string[]} list Existing list
- * @returns {string[]} List with parameters added
+ * @param node Function declaration
+ * @param list Existing list
+ * @returns List with parameters added
  */
-export function addArgumentsToList(node: FunctionDeclaration | ArrowFunctionExpression, list: string[]): string[] {
-  return addToList(
+export const addArgumentsToList = (node: FunctionDeclaration | ArrowFunctionExpression, list: string[]): string[] =>
+  addToList(
     node.params.reduce<string[]>((acc, param) => (isIdentifier(param) ? [...acc, param.name] : acc), []),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)